RELEASE_NOTES.md

```diff
@@ -1,5 +1,7 @@
 ## New
+- Display payment method icon in front of favorites.
```

Comment:
Nit: this can be written the same style as the rest

Reply:
Updated